src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.exceptions;
+
+public class BookNotFoundException extends RuntimeException {
+
+    public BookNotFoundException() {}
+
+    public BookNotFoundException(final String message, final Throwable cause) {
+        super(message, cause);
+    }
+
+    public BookNotFoundException(final String message) {
+        super(message);
+    }
+
+    public BookNotFoundException(final Throwable cause) {
```

Comment:
I don't think this constructor is necessary, leaving the constructor that accepts a custom message as a parameter is enough.

Reply:
I agree, but I need to implement it becasuse I'm using method reference in `Book b = bookRepository.findById(id).orElseThrow(BookNotFoundException::new);`

Reply:
You're right. I'll refactor it